test/common/schemaform/SaveFormLink.unit.spec.jsx

```diff
@@ -113,16 +115,16 @@ describe('Schemaform <SaveFormLink>', () => {
     // Find the login modal
     expect(modal).to.not.be.null;
   });
-  it('should call saveForm if logged in', () => {
-    saveFormSpy.reset(); // Just because it's good practice for a shared spy
+  it('should call saveInProgressForm if logged in', () => {
+    saveInProgressForm.reset(); // Just because it's good practice for a shared spy
```

Comment:
Could `beforeEach` be helpful for the reset logic here and for `document.querySelector` in the previous test?

Reply:
Yeah, we could do it in `beforeEach`, but it would then run before _every_ test, which is more than we need since we really only test it here.